atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -148,6 +151,11 @@ def run_in_worker(self):
         Exclude CSVs which already have a relatedImages section.
         """
         operator_manifest = self._get_operator_manifest()
+
+        if self._skip_all():
+            self.log.warning("skip_all defined not running on worker")
```

Comment:
```suggestion
            self.log.warning("'skip_all' defined; not running on worker")
```

I think this would be less confusing

Reply:
maybe just ? : 

skip_all defined, not running on worker

Reply:
ack

Reply:
updated